src/services/services.ts

```diff
@@ -1011,6 +1014,22 @@ module ts {
         isWriteAccess: boolean;
     }
 
+    export interface DocumentHighlights {
+        fileName: string;
+        highlightSpans: HighlightSpan[];
+    }
+
+    export class DocumentHighlightKind {
+        public static definition = "definition";
+        public static reference = "reference";
+        public static writtenReference = "writtenReference";
+    } 
+
+    export interface HighlightSpan {
+        textSpan: TextSpan;

```

Comment:
It's actually kind of odd that a HighlightSpan has a TextSpan instead of just being a TextSpan itself.